packages/react-core/__tests__/operations/groupby.test.js

```diff
@@ -1,12 +1,13 @@
 import { groupValuesByColumn } from '../../src/operations/groupby';
```

Comment:
I would suggest renaming this test file to `groupBy.test.js` (not `groupby`)

Reply:
It's a problem to change letter case in macOS because it isn't case sensitive 🤣 I think I know how to solve it using multiple commits hmmm, let me find out tomorrow 👌

Reply:
Done.